LibreHardwareMonitor/Utilities/HttpServer.cs

```diff
@@ -563,31 +569,100 @@ private async Task SendJsonAsync(HttpListenerResponse response, HttpListenerRequ
         response.Close();
     }
 
-    private async Task SendJsonSensorAsync(HttpListenerResponse response, Dictionary<string, object> sensorData)
+    private string GeneratePrometheusResponse(Node node)
     {
-        // Convert the JObject to a JSON string
-        string responseContent = System.Text.Json.JsonSerializer.Serialize(sensorData);
-        byte[] buffer = Encoding.UTF8.GetBytes(responseContent);
-
-        // Add headers and set content type
-        response.AddHeader("Cache-Control", "no-cache");
-        response.AddHeader("Access-Control-Allow-Origin", "*");
```

Comment:
Please add this back to SendJsonSensorAsync.

Reply:
They were removed in a different PR [here](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/commit/7063f5e0ae20c7b0e43de18bffb46fdd69c6d67d)
I added them there because of this. I will add them to SendResponseAsync.

Reply:
Maybe I am overthinking it, duplicating 2 lines that are easy to read shouldn't be a problem.